var/spack/repos/builtin/packages/py-scipy/package.py

```diff
@@ -96,10 +96,11 @@ def setup_build_environment(self, env):
         if platform.mac_ver()[0][0:2] == '11':
             env.set('MACOSX_DEPLOYMENT_TARGET', '10.15')
 
+    def flag_handler(self, name, flags):
+        return self.spec['py-numpy'].package.flag_handler(name, flags)
```

Comment:
Same question here, is this something that's needed for most packages that depend on numpy, or only scipy?

Reply:
Only if they compile fortran.

Reply:
And then, only sometimes.  py-scikit-learn doesn't need it, py-scipy does for linking lapack.